vars/liskBuild.groovy

```diff
@@ -27,15 +54,15 @@ def call(Map params) {
 			liskReleaseName = "lisk-${liskVersion}.tgz"
 			latestVersion = true
 		}
-		liskBuildDir = "lisk-build-${liskBuildBranch}-${Math.abs(new Random().nextInt() % 1000) + 1}"
+		liskBuildDir = "${WORKSPACE}/lisk-build-${liskBuildBranch}-${Math.abs(new Random().nextInt() % 1000) + 1}"
 	}
 	dir(liskBuildDir) {
 		git url: "https://github.com/LiskHQ/lisk-build.git", branch: liskBuildBranch
 		sh """
-		cp "../${liskDir}/release/${liskReleaseName}" "src/"
+		cp "${liskDir}/release/${liskReleaseName}" "src/"
 		bash build.sh -n "${buildNetwork}" -v "${liskVersion}"
 		"""
 	}
 
-	return [version: liskVersion, latestVersion: latestVersion, network: buildNetwork, file: "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz"]
+	return [version: liskVersion, latestVersion: latestVersion, network: buildNetwork, file: "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz", unversionedFile: "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz"]
```

Comment:
unversionedFile: "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz"

This should be either: 

- unversionedFile: "${liskBuildDir}/release/lisk-Linux-x86_64.tar.gz"
- nonVersionedFile: "${liskBuildDir}/release/lisk-Linux-x86_64.tar.gz"

depending on how the Jenkins job is going to get at it